static/js/lib/embed.js

```diff
@@ -59,6 +59,19 @@ export const loadEmbedlyPlatform = () => {
     const script = document.getElementsByTagName("script")[0]
     // $FlowFixMe
     script.parentNode.insertBefore(el, script)
+
+    // $FlowFixMe
+    embedly("on", "card.rendered", function(iframe) {
```

Comment:
I think you can remove the `$FlowFixMe` if you do `window.embedly` instead (worked locally for me anyhow)